ansible/roles/logstash-job-openshift/defaults/main.yaml

```diff
@@ -15,7 +15,3 @@ configs_dir: "{{ role_path }}/files/"
 # Define certs_dir to add certificates to the job
 # certs_dir: "{{ role_path }}/files/"
 
-
```

Comment:
the defaults should still be present. please don't delete

Reply:
added those defaults values back